package.json

```diff
@@ -44,31 +44,29 @@
   "author": "Anton Golub <a.golub@qiwi.com>",
   "license": "MIT",
   "dependencies": {
-    "tslib": "^2.2.0"
+    "tslib": "^2.4.0"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
-    "@semantic-release/github": "^7.2.3",
-    "@semantic-release/npm": "^7.1.3",
-    "@types/jest": "^26.0.23",
-    "@types/jest-json-schema": "^2.1.3",
-    "coveralls": "^3.1.0",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^8.0.6",
+    "@semantic-release/npm": "^9.0.1",
+    "@types/jest": "^29.2.0",
+    "@types/jest-json-schema": "^6.1.1",
+    "coveralls": "^3.1.1",
     "dts-generator": "^3.0.0",
-    "eslint-plugin-typescript": "^0.14.0",
+    "eslint": "^8.26.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-qiwi": "^1.17.8",
     "esm": "^3.2.25",
-    "flowgen": "1.14.1",
-    "jest": "^27.0.4",
+    "flowgen": "1.20.1",
+    "jest": "^29.2.2",
     "lodash": "^4.17.21",
-    "replace-in-file": "^6.2.0",
-    "semantic-release": "^17.4.3",
-    "ts-jest": "27.0.4",
-    "tslint": "^6.1.3",
-    "tslint-config-qiwi": "^1.7.1",
-    "tslint-config-standard": "^9.0.0",
-    "typedoc": "^0.20.36",
-    "typescript": "^4.3.2",
-    "typescript-eslint-parser": "^22.0.0"
+    "replace-in-file": "^6.3.5",
+    "semantic-release": "^19.0.5",
```

Comment:
`npx zx-semrel` means that we can remove semantic-release and all its plugins of dependencies